config/initializers/jsroutes.rb

```diff
@@ -0,0 +1,3 @@
+JsRoutes.setup do |config|
```

Comment:
What do you think about commenting these lines? It may not seem so easy to understand at first glance.